core/src/main/scripts/importer/validateData.py

```diff
@@ -2601,12 +2678,15 @@ def interface(args=None):
     parser.add_argument('-v', '--verbose', required=False, action='store_true',
                         help='report status info messages in addition '
                              'to errors and warnings')
+    parser.add_argument('-r', '--relaxed', type=str, required=False, 
+                        default='false',
+                        help='option to enable relaxed mode for validator')                             
```

Comment:
I would prefer this description to be more specific on what errors in the data it will ignore. Maybe even a different name for this option, to clarify which requirements of the file format are being relaxed. Something like `--no-clinical-definitions`, maybe?